src/stores/connectors/eth1/info.go

```diff
@@ -0,0 +1,12 @@
+package eth1
+
+import (
+	"context"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	"github.com/consensys/quorum-key-manager/src/stores/entities"
+)
+
+func (c Connector) Info(_ context.Context) (*entities.StoreInfo, error) {
```

Comment:
Do we have an issue about implementing this?

Reply:
We just didnt implemented yet

Reply:
Creata ticket to implement info() method